modules/rosa-mobb-prereq-checklist.adoc

```diff
@@ -0,0 +1,121 @@
+
+// Module included in the following assemblies:
+//
+// * rosa_planning/rosa-sts-aws-prereqs.html
+
+
+:_content-type: PROCEDURE
+[id="rosa-mobb-prereq-checklist_{context}"]
+= Prerequisites checklist to deploy a ROSA classic cluster 
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-15
+//---
+//date: '2023-07-27'
+//title: Prerequisites Checklist to Deploy ROSA Cluster with STS 
+//tags: ["ROSA", "STS"]
+//authors:
+//  - Byron Miller
+//  - Connor Wooley
+//  - Diana Sari
+//---
+
+This is a checklist of prerequisites needed to spin up a classic [product-title] cluster with links:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html[STS]. 
+
+[NOTE] 
+==== 
+This is a high level checklist and your implementation can vary. 
+====
+
+Before running the installation process, make sure that you deploy this from a machine that has access to:
+- The API services for the cloud to which you provision.
+- Access to `api.openshift.com` and `sso.redhat.com`. 
+- The hosts on the network that you provision.
+- The internet to obtain installation media.
+
+== Accounts and CLIs Prerequisites
+
+Accounts and CLIs you must install to deploy the cluster.
+
+=== AWS account:
+  - Gather the following details:
+      - AWS IAM User
```

Comment:
indents not working right in preview

![image](https://github.com/openshift/openshift-docs/assets/2488346/21bd0645-35ab-4c7a-a5f3-9b14b5e9ff2b)